docs-src/cli/README.md

```diff
@@ -0,0 +1,234 @@
+---
+id: readme
+title: temporal readme
+sidebar_label: Install CLI
+description: How to install and configure Temporal CLI.
+tags:
+    - cli
+---
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/temporalio/cli.svg)](https://pkg.go.dev/github.com/temporalio/cli)
+[![ci](https://github.com/temporalio/cli/actions/workflows/ci.yml/badge.svg)](https://github.com/temporalio/cli/actions/workflows/ci.yml)
+
+> ⚠️ Temporal CLI's API is still subject to change. ⚠️
+
+The Temporal CLI is a tool for interacting with Temporal from the command line.
+Temporal CLI functions as a distribution of the Temporal Server and Web UI that runs as a single process with zero runtime dependencies.
+The tool supports persistence to disk and in-memory mode through SQLite.
+
+## Installation
```

Comment:
Note that there's an entire page in the docs that explains how to install the CLI, you might want to inline or link it.